### ISSUE TITLE
Latest merge breaks CI builds

### ISSUE THREAD

**Issue by Zertsov** [grafana/dashboard-linter#136]
We have a build that pulls this repo, and the latest merge here broke the builds due to a `go.mod` replace directive being added here: https://github.com/grafana/dashboard-linter/pull/128/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R77

The error in question:
```
go: github.com/grafana/dashboard-linter@latest (in github.com/grafana/dashboard-linter@v0.0.0-20230621171119-d02699fb32c2):
The go.mod file for the module providing named packages contains one or
more replace directives. It must not contain directives that would cause
it to be interpreted differently than if it were the main module.
```

**Comment by piotrminkina** [comment]
I have the same problem. 

```console
$ go version
go version go1.19.8 linux/amd64
$ cat /etc/alpine-release 
3.17.3
```

**Comment by rgeyer** [comment]
Thanks for reporting this!, it looks to be resolved in #138 which I'll review and get merged ASAP.

This regression is fully on me. I saw that the CI broke, when I merged #128, but then ADHD brain kicked in and I totally forgot about it! 😩
